Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'rubocop'
 gem 'prettier'
 gem 'pry'
 gem 'iodine'
+gem 'faraday'
```

Comment:
Since this is an application (and not a gem), we generally should have our Gemfile.lock file checked into our repository. I bring it up because I would have expected to see a corresponding modification of the Gemfile.lock.

Reply:
@swcraig I added it to this PR, I added that file to .gitignore at some point but it's removed now